openupgrade_scripts/readme/ROADMAP.rst

```diff
@@ -0,0 +1,778 @@
++--------------------------------------------+-------------------------------------------------+
+|website_sale_delivery                       |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_digital                        |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_management                     |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_product_configurator           |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_slides                         |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_stock                          |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_stock_product_configurator     |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sale_wishlist                       |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_slides                              |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_slides_forum                        |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_slides_survey                       |                                                 |
++--------------------------------------------+-------------------------------------------------+
+|website_sms                                 |                                                 |
++--------------------------------------------+-------------------------------------------------+
+| |del| website_theme_install                | Merged into website                             |
++--------------------------------------------+-------------------------------------------------+
+|website_twitter                             |                                                 |
++--------------------------------------------+-------------------------------------------------+
+
+OCA modules
++++++++++++
+
+Here you will find the coverage of OpenUpgrade for other OCA modules that has
```

Comment:
This section is not maintained at all, so better to remove it, and link with current discussion at #2661

Reply:
Hi @pedrobaeza. 
- I just copied the file, not changed the content.
- AFAIK, this part is for Odoo features that moved in OCA modules (or the inverse). See, for the V12 coverage file : https://doc.therp.nl/openupgrade/modules110-120.html#oca-modules

Reply:
Yes, that's why I'm saying to remove it from both parts.